### PR TITLE
Clarify pointer arguments in more daemons

### PR DESCRIPTION
**Pull request by kgaillot**
This takes care of pacemaker-attrd and pacemaker-execd

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , here's another low-priority follow-up. No rush

File: daemons/attrd/attrd_attributes.c [hunk 1]
```diff
@@ -102,16 +102,16 @@ GHashTable *attributes = NULL;
  * \internal
  * \brief Create an XML representation of an attribute for use in peer messages
  *
- * \param[in] parent       Create attribute XML as child element of this element
- * \param[in] a            Attribute to represent
- * \param[in] v            Attribute value to represent
- * \param[in] force_write  If true, value should be written even if unchanged
+ * \param[in,out] parent      Create attribute XML as child element of this
```
Comment (nrwahl2) [inline daemons/attrd/attrd_attributes.c:105]: Did you mean to drop "element"?

File: daemons/attrd/attrd_attributes.c [hunk 2]
```diff
@@ -102,16 +102,16 @@ GHashTable *attributes = NULL;
...
+ * \param[in]     a           Attribute to represent
+ * \param[in]     v           Attribute value to represent
+ * \param[in]     force_write If true, value should be written even if unchanged
  *
```
Comment (nrwahl2) [inline daemons/attrd/attrd_attributes.c:109]: ^ if you feel like keeping two spaces after `force_write` for consistency. Wouldn't bother unless you're updating this for another reason

File: daemons/attrd/attrd_corosync.c [hunk 3]
```diff
@@ -249,9 +250,9 @@ record_peer_nodeid(attribute_value_t *v, const char *host)
 }
 
 static void
-update_attr_on_host(attribute_t *a, crm_node_t *peer, xmlNode *xml, const char *attr,
-                    const char *value, const char *host, bool filter,
-                    int is_force_write)
+update_attr_on_host(attribute_t *a, const crm_node_t *peer, xmlNode *xml,
```
Comment (nrwahl2) [inline daemons/attrd/attrd_corosync.c:253]: I believe `xml` can be const now

File: daemons/attrd/attrd_corosync.c [hunk 4]
```diff
@@ -316,7 +317,7 @@ update_attr_on_host(attribute_t *a, crm_node_t *peer, xmlNode *xml, const char *
 }
 
 static void
-attrd_peer_update_one(crm_node_t *peer, xmlNode *xml, bool filter)
+attrd_peer_update_one(const crm_node_t *peer, xmlNode *xml, bool filter)
```
Comment (nrwahl2) [inline daemons/attrd/attrd_corosync.c:320]: `xml` can be const

File: daemons/attrd/attrd_corosync.c [hunk 5]
```diff
@@ -362,7 +363,7 @@ attrd_peer_update_one(crm_node_t *peer, xmlNode *xml, bool filter)
 }
 
 static void
-broadcast_unseen_local_values(crm_node_t *peer, xmlNode *xml)
+broadcast_unseen_local_values(xmlNode *xml)
```
Comment (nrwahl2) [inline daemons/attrd/attrd_corosync.c:366]: The `xml` argument isn't used in the function

**Comment by nrwahl2** [comment]
That's all I've got :)

File: daemons/attrd/attrd_attributes.c (hunk 1 above)
Comment (kgaillot) [reply daemons/attrd/attrd_attributes.c:105]: Yes, to keep it under 80 characters

File: daemons/attrd/attrd_attributes.c (hunk 2 above)
Comment (kgaillot) [reply daemons/attrd/attrd_attributes.c:109]: Also to keep it under 80

File: daemons/attrd/attrd_corosync.c (hunk 3 above)
Comment (kgaillot) [reply daemons/attrd/attrd_corosync.c:253]: Thanks

File: daemons/attrd/attrd_corosync.c (hunk 4 above)
Comment (kgaillot) [reply daemons/attrd/attrd_corosync.c:320]: There's an xml_remove_prop() in this one

File: daemons/attrd/attrd_corosync.c (hunk 5 above)
Comment (kgaillot) [reply daemons/attrd/attrd_corosync.c:366]: Thanks

**Comment by kgaillot** [comment]
Updated attrd commit per review

**Pull request merged**